.github/workflows/release-please.yaml

```diff
@@ -202,3 +202,8 @@ jobs:
           TAG=${{ needs.release.outputs.tag_name }}
           BRANCH=$(echo "$TAG" | sed -e 's/\.0$/.x/')
           git push origin refs/tags/"$TAG"^{commit}:refs/heads/"$BRANCH"
+
+  update-demo-index:
+    name: Deploy Demo Version Index
+    needs: [appspot]
+    uses: ./.github/workflows/demo-version-index.yaml
```

Comment:
I might be misremembering, but don't we need a specific trigger for this in the target workflow?

Reply:
It is not necessary, because we are using needs, so we wait for all the appspot tasks to finish before running this one.